bleak/backends/bluezdbus/client.py

```diff
@@ -492,6 +493,16 @@ async def _acquire_mtu(self) -> None:
         """
         # This will try to get the "best" characteristic for getting the MTU.
         # We would rather not start notifications if we don't have to.
+        if BlueZFeatures.has_mtu_property:
```

Comment:
I would rather not try to populate this for BlueZ since there isn't a way to get the real value. Users can just use the per-characteristic property instead.

Reply:
Since we are effectively picking an arbitrary characteristic already, I figured it didn't make much difference.

Reply:
That was just a hack that was better than nothing from before we had a better option. I would rather not try to continue to support it since there is a better option now.